python/grass/jupyter/utils.py

```diff
@@ -225,7 +225,13 @@ def save_vector(name, geo_json):
 
 
 def get_number_of_cores(requested, env=None):
-    """Get the number of cores to use for multiprocessing."""
+    """Get the number of cores to use for multiprocessing.
+
+    :param int requested: Desired number of cores.
+    :param dict env: Optional environment dictionary with an "NPROCS" key.
```

Comment:
I know this is confusing, but this `env` doesn't contain `NPROCS`
```suggestion
    :param dict env: Optional process environment.
```